CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
-## 1.1.5
+## 1.2.0
 
 - Fix depending on a fully generated builder from a manually maintained builder.
+- Pick up fields on implemented interfaces, so you don't have to @override them.
+- Add BuiltValue annotation for specifying settings.
```

Comment:
Reading just this changelog without the example isn't very helpful. Yet, users want to learn about new things from the CHANGELOG. 

#### Option 1: build changelogs that include example usage, [like the Dart SDK does](https://github.com/dart-lang/sdk/blob/master/CHANGELOG.md#1240---12-06-2017), for example.

* Pros: Right there in the changelog. Doesn't need to update because in the changelog, example clearly attaches to a point in time.
* Cons: More work. Can make CHANGELOGs a bit unwieldy.

#### Option 2: add a canonical example or two to README.md and link to the section from here

* Pros: Example usage is easy to find and read. It's just one example so clear to know what to look at (contrast with many files in the `example/` dir). Way to advertise features to new users.
* Cons: Needs updating. Breaking changes to the example won't get auto-detected by analyzer.

#### Option 3: link from changelog to example files

* Pros: Least amount of immediate work. 
* Cons: Links can get stale or broken.